scrapers/ZexyVR.yml

```
@@ -0,0 +1,72 @@
+name: "ZexyVR"
+performerByURL:
+  - action: scrapeXPath
+    url:
+      - zexyvr.com/models
+    scraper: performerScraper
+sceneByURL:
+  - action: scrapeXPath
+    url:
+      - zexyvr.com/videos
+    scraper: sceneScraper
+xPathScrapers:
+  performerScraper:
+    common: 
+      $data: //div[@class="col-12 col-lg-7"]
+    performer:
+      Name: $data/h1/text()
+      Gender: 
+        fixed: female
+      Birthdate: 
+        selector: $data//li[normalize-space(text())="Birth date:"]/b[1]/text()
+        postProcess:
+          - parseDate: Jan 2, 2006
+      HairColor: $data//li[normalize-space(text())="Hair color:"]/b[1]/text()
+      EyeColor: $data//li[normalize-space(text())="Eye color:"]/b[1]/text()
+      Height: 
+        selector: $data//li[normalize-space(text())="Height:"]/b[1]/text()
+        postProcess:
+          - replace: 
+            - regex: " cm$"
+              with: ""
+      Weight: 
+        selector: $data//li[normalize-space(text())="Weight:"]/b[1]/text()
+        postProcess:
+          - replace: 
+            - regex: " kg$"
+              with: ""
+      Measurements: $data//li[normalize-space(text())="Bra size:"]/b[2]/text()
+      Image: 
+        selector: //img[@class="m-auto img-fluid"]/@src
+        postProcess:
+          - replace:
+            - regex: \?.*$
+              with: ""
+  sceneScraper:
+    common:
+      $info: //div[@class="row pt-3"]
+      $textBlob: //div[@class="row pt-3"]//p[@class="text-muted"]
+    scene:
+      Title: $info//h2/text()
+      Details: $info//p[2]/text()
+      Date: 
+        selector: $textBlob/text()[2]
+        postProcess:
+          - replace:
+            - regex: ^.*Released on 
+              with: ""
+          - parseDate: Jan 2, 2006
+      Studio:
+        Name:
+          fixed: ZexyVR
+      Tags: 
+        Name: $textBlob/a[@class="btn-link font-weight-light"]/text()
+      Performers:
+        Name: $textBlob/a[1]/text()
```

it seems that all scenes have a single performer but why not use a safer selector just in case ?
```suggestion
        Name: $textBlob/a[contains(@href,"/models/")]/text()
```